doc/articles/features/file-management.md

```diff
@@ -30,13 +47,13 @@ Note that for WebAssembly in particular, the `await localFolder.CreateFolderAsyn
 
 Note that you can view the content of the **IndexedDB** in the Application tab of your browser, in the **Storage / IndexedDB** section.
 
-## Support for StorageFile.GetFileFromApplicationUriAsync
+## Support for `StorageFile.GetFileFromApplicationUriAsync`
 
 Uno supports the ability to get package files using the [`StorageFile.GetFileFromApplicationUriAsync`](https://docs.microsoft.com/en-us/uwp/api/windows.storage.storagefile.getfilefromapplicationuriasync).
 
 Support per platform may vary:
-- On Android, iOS/macOS, the file is available immediately as it is part of the installed package.
-- On WebAssembly, the requested file is part of the remote package and is downloaded on demand to avoid increasing the initial application payload size. The file is then stored in the IndexedDB of the browser.
+- On non-WebAssembly targets, the file is available directly as it is a part of the installed package.
+- On WebAssembly, the requested file is part of the application package on the remote server, and is downloaded on demand to avoid increasing the initial application payload size. After it is requested for the first time, the file is then stored in the browser IndexedDB.
```

Comment:
```suggestion
- On WebAssembly, the requested file is part of the application package on the remote server and is downloaded on demand to avoid increasing the initial application payload size. After it is requested for the first time, the file is then stored in the browser IndexedDB.
```